.github/workflows/release.yml

```diff
@@ -2,7 +2,7 @@ name: Release
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v\d+\.\d+\.\d+'
```

Comment:
I was reading https://stackoverflow.com/questions/61891328/trigger-github-action-only-on-new-tags and wonder if there's any reason to specify this pattern at all, and not just use `'*'`?